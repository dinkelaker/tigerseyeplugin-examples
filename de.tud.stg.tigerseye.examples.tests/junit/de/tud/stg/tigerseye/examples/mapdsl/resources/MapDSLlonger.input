    def hanspeter = [Integer,String: 1="hans",2="peter"]    
    
    println hanspeter
    
    def doubleint = [Double , Integer : 0=0,1=1,2=2,3=3, 4=4,5=5,6=6,7=7,8=8,9=9,10=10]
    println doubleint
 
    def hans = [String, Object : "name"="Hans", "lastname"="HÃ¤user", "age"=21 , "married"=false]    
    println "hans ist $hans"
    
    def peter = [String, Object : "name"= "Peter", "lastname"="Bauer", "age"=45, "married"=true]    
    println "peter ist $peter"
    
    hans.putAll(peter) 
    assert hans.equals(peter)