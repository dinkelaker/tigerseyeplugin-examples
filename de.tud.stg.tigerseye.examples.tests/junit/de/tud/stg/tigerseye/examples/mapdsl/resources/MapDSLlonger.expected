    
    def hanspeter = buildMap(
Integer.class,
String.class,
[
buildEntry(
1,
"hans"),
buildEntry(
2,
"peter")
] as Entry[])   
    
    println hanspeter
    
    def doubleint = buildMap(
Double.class,
Integer.class,
[
buildEntry(
0,
0),
buildEntry(
1,
1),
buildEntry(
2,
2),
buildEntry(
3,
3),
buildEntry(
4,
4),
buildEntry(
5,
5),
buildEntry(
6,
6),
buildEntry(
7,
7),
buildEntry(
8,
8),
buildEntry(
9,
9),
buildEntry(
10,
10)
] as Entry[])
    println doubleint
 
    def hans = buildMap(
String.class,
Object.class,
[
buildEntry(
"name",
"Hans"),
buildEntry(
"lastname",
"HÃ¤user"),
buildEntry(
"age",
21),
buildEntry(
"married",
false)
] as Entry[])   
    println "hans ist $hans"
    
    def peter = buildMap(
String.class,
Object.class,
[
buildEntry(
"name",
"Peter"),
buildEntry(
"lastname",
"Bauer"),
buildEntry(
"age",
45),
buildEntry(
"married",
true)
] as Entry[])   
    println "peter ist $peter"
    
    hans.putAll(peter) 
    assert hans.equals(peter)
